cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -353,12 +354,13 @@ teeTraceChainTip
   => TraceOptions
   -> MVar (Maybe (WithSeverity (ChainDB.TraceEvent blk)), Integer)
   -> Trace IO Text
+  -> Trace IO Text
   -> Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
-teeTraceChainTip TracingOff _ _ = nullTracer
-teeTraceChainTip (TracingOn trSel) elided tr =
+teeTraceChainTip TracingOff _ _ _ = nullTracer
+teeTraceChainTip (TracingOn trSel) elided trTrc trMet =
```

Comment:
Why are we adding an additional parameter to `teeTraceChainTip` when both parameters accept the same tracer?  Can't we move `appendName` into `teeTraceChainTip`?

Reply:
@Jimbo4350, it's to make it a bit more apparent at the top level of the call chain that we're posting metrics to two different namespaces.